go/interpreter/lfvm/instructions.go

```diff
@@ -250,7 +250,10 @@ func opSstore(c *context) error {
 	cost := tosca.Gas(0)
 	if c.isAtLeast(tosca.R09_Berlin) &&
 		c.context.AccessStorage(c.params.Recipient, key) == tosca.ColdAccess {
-		cost += 2100
+		if err := c.useGas(ColdSloadCostEIP2929); err != nil {
```

Comment:
The code before the refactor may have been already aware of this issue. 
please revert this lines and document why cost is accumulated but not substracted yet.